docs/ci.md

```diff
@@ -0,0 +1,28 @@
+# Continuous Integration
+
+## GitHub Workflows
+
+The GitHub Workflows under [`.github/workflows/][workflows] contain jobs that
+are started when Pull-Requests are created or updated. Some of the jobs can
+build container-images for multiple architectures. Not everyone or all
+environmens wants to run the build tests for all platforms. The workflows can
+be configured to select platforms that the `docker/setup-buildx-action`
+supports.
+
+For this configuration, a new Secret should be created in the GitHub
+Settings of the repository. 'Normal' environment variables seem not possible.
+
+An example of the GitHub Secret that will build the container-images on AMD64,
+and both 32-bit and 64-bit Arm platforms:
```

Comment:
`Arm` to `ARM` ?

Reply:
https://www.arm.com/ uses Arm (I think that the new capitalization was introduced with some rebranding a while ago)